src/utils/holoflowKits.ts

```diff
@@ -0,0 +1,12 @@
+import { LiveSelector, MutationObserverWatcher } from '@holoflows/kit'
+
+export const untilElementAvailable: {
```

Comment:
Why using this file name? That's confusing, consider rename it to `untilElementAvailable.ts`

Reply:
It's now in a folder named holoflowKits

Reply:
The current name is still confusing. I think you're misusing the term `holoflows`

Reply:
emmm I thought that this maybe a missing feature of holoflowKits so it's in this folder...

Reply:
fixed: rearrange